sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -599,10 +610,13 @@ object JdbcUtils extends Logging with SQLConfHelper {
           stmt.setTimestamp(pos + 1, toJavaTimestamp(instantToMicros(row.getAs[Instant](pos))))
       } else {
```

Comment:
since we are touching the code here, let's make it right. I think for ntz type, its value is always `java.time.LocalDateTime`, no matter `datetimeJava8ApiEnabled` is true or not, right? @MaxGekk @gengliangwang

Reply:
Yes, TimestampNTZ is independent of the configuration.

Reply:
Updated.